Bio/PDB/Entity.py

```diff
@@ -173,9 +175,11 @@ def id(self, value):
             return
         if self.parent:
             if value in self.parent.child_dict:
-                raise ValueError(
-                    f"Cannot change id from `{self._id}` to `{value}`."
-                    f" The id `{value}` is already used for a sibling of this entity."
+                # See issue 1551 for details on the downgrade.
+                warnings.warn(
+                    f"The id `{value}` is already used for a sibling of this entity. "
+                    f"Changing id from `{self._id}` to `{value}` might create access "
+                    "inconsistencies to children of the parent entity."
```

Comment:
Use ``from Bio import BiopythonWarning`` to make it slightly easier for people to silence this if they really want to do this?